src/views/preview/project-view.jsx

```diff
@@ -29,6 +29,8 @@ const Meta = require('./meta.jsx');
 const sessionActions = require('../../redux/session.js');
 const navigationActions = require('../../redux/navigation.js');
 const previewActions = require('../../redux/preview.js');
+const commentsActions = require('../../redux/comments.js');
```

Comment:
Remove this and move `resetComments` and `addComment` to the projectCommentActions file. This will simplify how the consumer of a type of comment has to interact with the actions

Reply:
Done 8a33d1c /cc @kchadha